scripts/uberspace-setup-postgres

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Have you written it? Or is this copy+pasted from some uberspace docs?

Reply:
It’s basically this [Uberspace guide](https://lab.uberspace.de/guide_postgresql.html) on how to run Postgres, but in the form of a bash script as I was annoyed by having to set up everything over and over when creating new Uberspace accounts. I’m pretty sure there are a few cases it does not handle – but as long as you run it on a fresh Uberspace account, it should work just fine.